src/scout_apm/api/__init__.py

```diff
@@ -44,3 +44,50 @@ def tag(self, key, value):
         if self.span is not None:
             self.span.tag(key, value)
 
+
+class Transaction(ContextDecorator):
+    def __init__(self, name, tags={}):
+        self.name = name
+        self.tags = tags
+
+    def start(kind, operation, tags={}):
+        operation = kind + '/' + operation
+
+        tr = TrackedRequest.instance()
+        span = tr.start_span(operation=operation)
+        for key, value in tags.items():
+            tr.tag(key, value)
+        return span
+
+    def stop():
+        tr = TrackedRequest.instance()
+        tr.stop_span()
+        return False
```

Comment:
return `True` instead?

Reply:
I suppose - it was false since the `__exit__` function needs `False` returned. But when called directly, I agree `True` makes more sense. 

https://docs.python.org/3/reference/datamodel.html#with-statement-context-managers